.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
+dist
```

Comment:
Are you planning on removing `dist` from source control? Or just ignoring changes to it? To cut a release after this change, would we need to forcibly add the changes to the dist folder to the commit?

Reply:
It would be a major to remove dist at this point I think.

I just don't want PRs / git status to show it, untill explicitly added via `git add dist -f`.